### DISCUSSION TITLE
Not able to make it work in Lubuntu

### DISCUSSION THREAD

**Discussion by alejandro-zapeta** [kounoike/obs-virtualbg#104] [General]
let me share some context:

OBS: 27.2.3+dfsg1-1 (64bit)
Lubuntu: 22.04

i did the  `sudo dpkg -i obs-virtualbg-X.X.X-Linux.deb`

But i can't see the filters in the list of filters
![image](https://github.com/kounoike/obs-virtualbg/assets/2177278/82a3c51d-d09d-4e9c-8125-1103c2477136)

Am I missing something?
Thank you so much.

**Comment by Canuckfanatic80** [comment]
Greetings. I am also having this problem: I cannot see the plugin in the filters? I tried all kinds of things as you displayed here. I am using:

 Kernel: 5.15.0-126-generic x86_64 bits: 64 compiler: gcc v: 11.4.0 Desktop: MATE 1.26.0
    info: mate-panel wm: marco 1.26.0 vt: 7 dm: LightDM 1.30.0 Distro: Linux Mint 21.1 Vera
    base: Ubuntu 22.04 jammy
